### PR TITLE
add server.js

### PR DESCRIPTION
**Pull request by Git-Hannah**
Please review!

### CODE REVIEW

**Comment by DenSchmi81** [comment]
lgtm

File: package.json [hunk 1]
```diff
@@ -11,9 +14,11 @@
   "dependencies": {
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
+    "express": "^4.17.2",
+    "nodemon": "^2.0.15",
```
Comment (marwinburesch) [inline package.json]: This should be under `devDependencies`

File: server.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+app.get('/', (request, response) => {
+	response.send('Hello World!')
+  })
+
+
+  app.get('/todos/', (request, response) => {
+	  response.json([
+			  { id: 1, name: "First toDo"},
+		  { id: 2, name: "Second toDo"},
+	  ])
```
Comment (marwinburesch) [inline server.js]: Looks like something is off with your code formatting

File: server.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  })
+
+  app.listen(port, () => {
+	console.log(`Example app listening on port ${port}`)
```
Comment (marwinburesch) [inline server.js]: I'd change this text here just so it's different from the example

File: todos.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "name": "Marga"
+}
```
Comment (marwinburesch) [inline todos.json:3]: This seems like a change for the next PR :)

Comment (Git-Hannah) [reply todos.json:3]: yes.. I am still totally aware of this file which was part of my answer finding process :oD hihi

**Pull request merged**